WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageParentViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.viewmodel.pages
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import org.wordpress.android.R.string
+import org.wordpress.android.fluxc.Dispatcher
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.pages.PageItem
+import org.wordpress.android.ui.pages.PageItem.Action
+import org.wordpress.android.ui.pages.PageItem.Empty
+import javax.inject.Inject
+
+class PageParentViewModel
```

Comment:
It looks like `PageListViewModel` and `PageParentViewModel` are identical.  Can we use a single class for both?

Reply:
This is a WIP PR and this is just a non-functional boilerplate code. These classes will be modified later, I just created them so that the final PR is smaller.